src/pages/components/checkbox.mdx

```diff
@@ -1,15 +1,183 @@
 ---
 title: Checkbox
 description: >
-  Description will be here
+  A Checkbox lets a user select an option. It provides an easy way to make a choice, and helps
+  users compare options.
+package: '@zendeskgarden/react-forms'
+components:
+  - forms/src/elements/Checkbox.tsx
+  - forms/src/elements/common/Field.tsx
+  - forms/src/elements/common/Hint.tsx
+  - forms/src/elements/common/Label.tsx
+  - forms/src/elements/common/Message.tsx
 ---
 
-Content here
+<Usage>
+  <Use>
+    <ul>
+      <li>To select any number of options, including zero</li>
+      <li>As a standalone control to turn a single option on or off</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        To limit the user to selecting exactly one option from a set, use{' '}
+        <a href="../radio">Radios</a>
```

Comment:
Invalid href. Also check "Radios" wording against "Radio" page. Copy might need to be something like "use the _Radio_ component"